cyences_app_for_splunk/default/data/ui/views/cs_sophos_reports.xml

```diff
@@ -81,11 +81,11 @@
       <title>Endpoint Service Not Running</title>
       <table>
         <search>
-          <query>`cs_sophos` type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") $tkn_host$ $tkn_dhost$ $tkn_user$ 
-| stats count, latest(_time) as _time, values(src_ip) as src_ip, values(suser) as user latest(type) as type by host, dhost 
+          <query>index=main sourcetype="sophos*"  type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") host="*" location IN ("*") source IN ("*")
+| stats count, latest(_time) as _time, values(source_info.ip) as source_info.ip, values(source) as source latest(type) as type by host, location
 | where type=="Event::Endpoint::ServiceNotRunning"
-| `cs_human_readable_time_format(_time, "Service Disabled Since")` 
-| table host, dhost, src_ip, user, "Service Disabled Since"</query>
+| `cs_human_readable_time_format(_time, "Service Disabled Since")`
+| table host, location, source_info.ip, source, "Service Disabled Since"</query>
```

Comment:
Same comment for all the searches in this file:

Use the macro for the index and try to find the exact source type.
index=main sourcetype="sophos*" => cs_sophos sourcetype="sophos_events"

Do not required this filters => host="" location IN ("") source IN ("*")
instead use this token for the filter => $tkn_host$ $tkn_dhost$ $tkn_user$

rename source_info.ip as src_ip

values(source) as source => is this splunk's metadata field or event's field? if it's extracted from event then it also include the values of splunk's metadata field source